lambdas/service/app.py

```diff
@@ -1466,20 +1472,21 @@ def handle_manifest_generation_request():
     object_key = None
     token = query_params.get('token')
     if token is None:
-        object_key, presigned_url = service.get_cached_manifest(format_=format_,
-                                                                catalog=catalog,
-                                                                filters=filters)
-        if presigned_url is not None:
-            return 0, presigned_url
-    retry_url = app.self_url()
+        object_key, manifest = service.get_cached_manifest(format_=format_,
+                                                           catalog=catalog,
+                                                           filters=filters)
+        if manifest is not None:
+            return 0, manifest
     async_service = AsyncManifestService()
     try:
-        return async_service.start_or_inspect_manifest_generation(retry_url,
-                                                                  format_=format_,
-                                                                  catalog=catalog,
-                                                                  filters=filters,
-                                                                  token=token,
-                                                                  object_key=object_key)
+        wait_time, manifest = \
```

Comment:
the contribution guidelines indicates that we should refrain from using `\` (unless there is no other option). I think returning to `return asyc_service` might be better than using wrapped `()` or `\`. It seems that going back to the old way would be ok as there are no inspections performed on the wait time or manifest.

Reply:
Good catch. Early in development of this ticket the return values of `start_or_inspect_manifest_generation()` required special handling at this location, but now that the function has been modified we can just simply return the returned values as is.